app/src/main/java/com/depromeet/team6/presentation/ui/course/component/TransportTabMenu.kt

```diff
@@ -0,0 +1,94 @@
+package com.depromeet.team6.presentation.ui.course.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.depromeet.team6.presentation.model.course.LastTransportInfo
+import com.depromeet.team6.presentation.model.course.TransportCourseInfo
+import com.depromeet.team6.presentation.model.course.TransportType
+import com.depromeet.team6.ui.theme.defaultTeam6Colors
+import kotlinx.coroutines.launch
+
+@Composable
+fun TransportTabMenu(
+    availableCourses: List<LastTransportInfo>,
+    modifier: Modifier = Modifier
+) {
+    val tabItems = listOf("전체", "버스", "지하철")
```

Comment:
```suggestion
    val tabItems = immutableListOf("전체", "버스", "지하철")
```

바뀌는 항목들이 아니라면? 저희 스터디 때 이야기한 내용처럼 immutable한 list로 설정하는게 Stable해질 것 같아요!

Reply:
앗 그러게요 
저건 그냥 리소스파일 배열로 뺄게요!